client/src/main/scala/com.gu.contentapi.client/model/HttpResponse.scala

```diff
@@ -0,0 +1,14 @@
+package com.gu.contentapi.client.model
+
+import scala.concurrent.Future
+
+case class HttpResponse(body: Array[Byte], statusCode: Int, statusMessage: String)
+
+object HttpResponse {
+  val isValid = Set(200, 302) 
+  
+  def check: HttpResponse => Future[Array[Byte]] = {
```

Comment:
Is there a benefit for returning a `Function1` here instead of defining `check` as a regular method that accepts an `HttpResponse` and returns a `Future`?

Reply:
It saves a few keystrokes 😅